ZipBuilder/Tests/main.swift

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import XCTest
+
+import sem_versionsTests
+
+var tests = [XCTestCaseEntry]()
```

Comment:
Can you please add a note here stating this is required for Linux only? (I think that's the case but I'm not sure..)

Reply:
I renamed the file back to `LinuxMain.swift` as it was originally generated. TBH, I don't feel an expert here so I accept suggestions 😄